WordPress/Classes/Services/ReaderTopicService.m

```diff
@@ -473,14 +473,16 @@ - (void)tagTopicForTagWithSlug:(NSString *)slug success:(void(^)(NSManagedObject
     }];
 }
 
-- (void)toggleFollowingForSite:(ReaderSiteTopic *)siteTopic success:(void (^)(void))success failure:(void (^)(NSError *error))failure
+- (void)toggleFollowingForSite:(ReaderSiteTopic *)siteTopic
+                       success:(void (^)(BOOL follow))success
+                       failure:(void (^)(BOOL follow, NSError *error))failure
 {
     NSError *error;
     ReaderSiteTopic *topic = (ReaderSiteTopic *)[self.managedObjectContext existingObjectWithID:siteTopic.objectID error:&error];
     if (error) {
         DDLogError(error.localizedDescription);
         if (failure) {
-            failure(error);
+            failure(true, error);
```

Comment:
Should this be `follow` instead of `true`?

Reply:
For future reference: see the conversation above.